src/test/java/tec/uom/se/format/SymbolMapTest.java

```diff
@@ -52,7 +52,7 @@
    */
   @Test
   public void testSymbolMap() {
-    SymbolMap symbols = SymbolMap.of(ResourceBundle.getBundle(BUNDLE_NAME, Locale.getDefault()));
+    SymbolMap symbols = SymbolMap.of(ResourceBundle.getBundle(BUNDLE_NAME, Locale.ROOT));

```

Comment:
Yeeah, and it won't make the test fail if the user tries to run the tests on Japan or some of these other countries where there's a different symbol for some units :D